DAO.sol

```diff
@@ -548,7 +556,7 @@ contract DAO is DAOInterface, Token, TokenSale {
             // p.proposalDeposit should be zero here
             if (this.balance < p.proposalDeposit)
```

Comment:
This overflow check should use `sumOfProposalDeposits` instead of `p.proposalDeposit`.